fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -415,7 +424,7 @@ def _threshold_optimization_for_simple_constraints(self, sensitive_features, lab
             # Determine probability of current sensitive feature group based on data.
             p_sensitive_feature_value = len(group) / n
 
-            roc_convex_hull = _tradeoff_curve(
+            metrics_curve_convex_hull = _tradeoff_curve(
```

Comment:
This is not always the roc_curve's convex hull. 

It only coincides with the roc_curve if y is "true_positive_rate" and x is "false_positive_rate".